docs/reference-guides/core-blocks.md

```diff
@@ -36,7 +36,7 @@ Add a user’s avatar. ([Source](https://github.com/WordPress/gutenberg/tree/tru
 -	**Supports:** align, color (~~background~~, ~~text~~), spacing (margin, padding), ~~alignWide~~, ~~html~~
 -	**Attributes:** isLink, linkTarget, size, userId
 
-## Reusable block
+## Pattern
```

Comment:
I think we'll have two blocks called Pattern after this 🤔 

It might be ok, but another option is to also rename the other pattern block to something else. ('Pattern placeholder'?)

The reason why I'd rename the other one is because it can't really be inserted and isn't visible in the UI, so I think its name is less important.

Reply:
Fixed.